modules/tinymce/src/plugins/table/main/ts/selection/TableSelection.ts

```diff
@@ -6,28 +6,33 @@
  */
 
 import { CellOpSelection, Selections, TableSelection } from '@ephox/darwin';
-import { Arr, Optionals } from '@ephox/katamari';
+import { Arr, Fun, Optionals } from '@ephox/katamari';
 import { TableLookup } from '@ephox/snooker';
 import { Attribute, Compare, SelectorFind, SugarElement, SugarElements, SugarNode } from '@ephox/sugar';
 
 import { ephemera } from './Ephemera';
 
-const getSelectionStartCellFallback = (start: SugarElement<Node>) =>
-  TableLookup.table(start).bind((table) =>
+const getSelectionCellFallback = (element: SugarElement<Node>, takeLast: boolean) =>
```

Comment:
The `takeLast` argument here is never used as it's value is always `false` and doesn't really make sense either as the cell fallback logic is used to move from a col element to a cell element, so it should always be the first row/cell we want.

- [ ] Revert `takeLast` logic

Reply:
You are right, it should have been true at one point. My bad, fixed that.

Reply:
Did you read what I said above? This logic is **incorrect** so it shouldn't have been fixed to use it and should have been removed. To re-iterate this handles the case where the selection ends up in an invalid location within a `<col>` element (Firefox IIRC was the only browser that did this). So when it gets in that state it should **always** be using the first cell as that's always the closest valid position to use. Using the last cell within a selection is definitely the wrong thing to be doing.

Reply:
I read it, but misunderstood what you referred to. My apologizes.